### PR TITLE
DOCS-7500 graph search

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Review by sancheztavo — APPROVED** [review]
But remember that this search won't be GA for a while yet.

File: modules/ROOT/pages/to-find-info.adoc [hunk 1]
```diff
@@ -34,7 +34,9 @@ You can combine search terms with the filters listed in All Types.
 
 You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
 
-Searches only apply to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+In Exchange Private Cloud Edition and GovCloud, searches only apply to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/to-find-info.adoc]:
```suggestion
In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/to-find-info.adoc]: Moving "only" to the proper place on the second one just sounds off. I'd check with Kim on this when you follow up.

File: modules/ROOT/pages/to-find-info.adoc [hunk 2]
```diff
@@ -80,7 +82,7 @@ starts with `mq` or `module`.
 
 Searching for a multiple word phrase shows assets that have every word in the phrase.
 
-If you separate search words with any character other than a space, Exchange
+If you separate search words with any non-alphanumeric character other than a space, Exchange
```
Comment (sarathecoubian) [inline modules/ROOT/pages/to-find-info.adoc]:
```suggestion
If you separate search words with any non-alphanumeric character other than a space, Anypoint Exchange
```

File: modules/ROOT/pages/to-manage-categories.adoc [hunk 3]
```diff
@@ -49,6 +49,12 @@ The following search terms return all assets marked as both System APIs and Sale
 
 `category:"API Type" = "System API" category:"Product" = "Salesforce"`
 
+== Search with Categories in Exchange Cloud
+
+Select a category such as "API Type" from the list, and select one or more values such as "System API". Exchange shows assets that have one of the selected values for the selected category.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/to-manage-categories.adoc]:
```suggestion
Select a category such as "API Type" from the list, and select one or more values such as "System API". Anypoint Exchange shows assets that have one of the selected values for the selected category.
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**